inst/@sym/heaviside.m

```diff
@@ -32,34 +33,41 @@
 %% @end group
 %% @end example
 %%
-%% By default, the value at zero is undefined:
+%% There are various conventions for 'heaviside(sym(0))'; this function
+%% returns (sym) 1/2 by default.
```

Comment:
I think you should just say "returns 1/2 by default" or maybe "returns the midpoint by default:" (and let the example show it is 1/2).

Other than that, this looks ready to go1